projects/packages/jetpack-mu-wpcom/src/features/block-theme-previews/modal.jsx

```diff
@@ -1,74 +0,0 @@
-import { Button, Modal } from '@wordpress/components';
-import { useDispatch, useSelect } from '@wordpress/data';
-import { __, sprintf } from '@wordpress/i18n';
-import React from 'react';
-import { getPreviewedThemeStylesheet } from './utils';
-
-import './modal.scss';
-
-/**
- * The modal that shows in the Site Editor the first time a user previews a block theme.
- *
- * @return {JSX.Element|null} The modal component if visible and in the Site Editor, otherwise null.
- */
-export default function BlockThemePreviewsModal() {
```

Comment:
Hmm, isn't this still used? Though I haven't been able to see the modal 🤔

Reply:
I removed the line that required `block-theme-previews.php` in a previous PR, so it's no longer used. I initially thought Core didn't have the modal either. 😅

Reply:
Ohhh... right right, I missed that fact. Hmm, I don't have strong objection though, if we want to make us close to Core, then I believe it's fine 🤔